.travis.yml

```diff
@@ -10,68 +10,22 @@ env:
   - GOSUMDB=https://sum.golang.org
 
 os:
-  - linux
   - linux-ppc64le
```

Comment:
suggest a TODO here to remove travis yml if/when github actions supports the remaining platforms...

Reply:
done